autoware_launch/launch/components/tier4_perception_component.launch.xml

```diff
@@ -102,6 +102,10 @@
       name="object_recognition_detection_object_merger_distance_threshold_list_path"
       value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/detection/object_merger/overlapped_judge.param.yaml"
     />
+    <arg
+      name="object_recognition_detection_object_merger_node_param_path"
+      value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/detection/object_merger/object_association_merger.param.yaml"
+    />
```

Comment:
We will also need this `object_recognition_detection_object_merger_node_param_path` passed down into `tier4_perception_launch/perception.launch.xml`, won't we?